utils/config.go

```diff
@@ -164,6 +164,7 @@ type Config struct {
 	DeletionDb             string         // directory of deleted account database
 	DiagnosticServer       int64          // if not zero, the port used for hosting a HTTP server for performance diagnostics
 	ErrorLogging           string         // if defined, error logging to file is enabled
+	EvmImpl                string         // evm implementation (aida/opera/tosca)
```

Comment:
Out of curiosity: what does choosing `opera` do? I dont see any code relevant to this option

Reply:
There will be several implementations of the EVM:
- "aida" - the one we had so far, implemented in Aida
- "opera" - a reference implementation implemented in Tosca importing Opera/Sonic code; this one is very similar to the `aida` version, but located in Tosca as a base-line implementation; it might also get used initially by the new client architecture until the "tosca" option is available;
- "tosca" - a widely Geth/Opera/Sonic independent implementation in Tosca